frontend/views/containers/proposals/PropositionsAllModal.vue

```diff
@@ -46,20 +49,30 @@ export default ({
       selectbox: {
         focused: false,
         selectedOption: 'Newest'
-      }
+      },
+      proposals: []
     }
   }),
+  async mounted () {
+    // const openProposals = Object.entries(this.currentGroupState.proposals)
+    //   .sort((a, b) => {
+    //     return compareISOTimestamps(a[1].meta.createdDate, b[1].meta.createdDate)
+    //   })
+    const key = `proposals/${this.ourUsername}/${this.currentGroupId}`
```

Comment:
Currently `/` characters are still allowed in usernames I think. This might precludes interpolation in SBP selectors like here.

Reply:
This isn't an SBP selector, it's just a key into a key-value database, so it's OK.